src/main/client/netcore.client.ftl

```diff
@@ -137,4 +137,22 @@ namespace io.fusionauth {
       return new DefaultRESTClient(host);
     }
   }
+
+  public class HttpClientBuilder : IRESTClientBuilder
+  {
+    public HttpClient HTTP_CLIENT;
+
+    public HttpClientBuilder(HttpClient httpClient)
+    {
+      HTTP_CLIENT = httpClient;
+    }
+
+    public IRESTClient build(string host)
+    {
+      if (HTTP_CLIENT.BaseAddress == null)
```

Comment:
If a `host` string is passed in, but the `BaseAddress` of the `HTTP_CLIENT` is already set, the parameter is effectively ignored. This might be an anti-pattern.